src/components/alert/alert.component.ts

```diff
@@ -0,0 +1,248 @@
+import { animateTo, stopAnimations } from '../../internal/animate';
+import { classMap } from 'lit/directives/class-map.js';
+import { getAnimation, setDefaultAnimation } from '../../utilities/animation-registry';
+import { HasSlotController } from '../../internal/slot';
+import { html } from 'lit';
+import { LocalizeController } from '../../utilities/localize';
+import { property, query } from 'lit/decorators.js';
+import { waitForEvent } from '../../internal/event';
+import { watch } from '../../internal/watch';
+import ShoelaceElement from '../../internal/shoelace-element';
+import SlIconButton from '../icon-button/icon-button';
```

Comment:
To show this fully working you'll need to convert icon-button to be scoped and only import the non-auto-registering module here.